### PR TITLE
Quickstart gradle fixes

### PR DESCRIPTION
**Pull request by barchetta**
* Fix dependencies 
* Use correct main class
* Enable Junit tests

### CODE REVIEW

File: examples/quickstarts/helidon-quickstart-mp/build.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 plugins {
     id 'java'
-    //id 'org.kordamp.gradle.jandex' version '0.5.0'
+    id 'org.kordamp.gradle.jandex' version '0.5.0'
```
Comment (aalmiray) [inline examples/quickstarts/helidon-quickstart-mp/build.gradle:19]: Version `0.6.0` is out, which makes sure the jandex file is placed automatically under `META-INF` just as its Maven counterpart does.

Comment (barchetta) [reply examples/quickstarts/helidon-quickstart-mp/build.gradle]: I've upgraded to 0.6.0

File: examples/quickstarts/helidon-quickstart-mp/build.gradle [hunk 2]
```diff
@@ -91,3 +80,17 @@ jar {
                )
   }
 }
+
+// This is a work-around for running unit tests.
+// Gradle places resource files under ${buildDir}/resources. In order for
+// beans.xml to get picked up by CDI it must be co-located with the classes.
+// So we move it before running tests.
+// In either case it ends up AOK in the final jar artifact
+task moveBeansXML {
```
Comment (aalmiray) [inline examples/quickstarts/helidon-quickstart-mp/build.gradle:89]: I'm not keen on using the move option but that's what the Gradle guide recommends

https://docs.gradle.org/current/userguide/working_with_files.html#sec:moving_files_example

**Comment by aalmiray** [comment]
LGTM. Ship it! 😏

File: examples/quickstarts/helidon-quickstart-mp/build.gradle (hunk 2 above)
Comment (barchetta) [reply examples/quickstarts/helidon-quickstart-mp/build.gradle:89]: Yes, I had a similar reaction.

**Pull request merged**